File: Java/UberX.java
class UberX extends Car {
  String brand;
  String model;
   public UberX(String licence, Drver driver, String brand, String model){
     super(licence, driver);
     this.brand = brand;
     this.model = model;
   }
   @Override
   void printDataCar() {
     super.printDataCar();
     System.out.println("\tModelo: " + this.model + "\n\tBrand:" + this.brand);
   }
}

File: PHP/uberX.php
<?php
require_once('car.php');
require_once 'drver.php';
class UberX extends car{
  public $brand;
  public $model;
  public function __construct(String $licence, Drver $driver, $brand, $model)
  {
    parent::__construct($licence,$driver);
    $this->brand = $brand;
    $this->model = $model;
  }
  public function printDataCar(){
    parent::printDataCar();
    echo "<br>Brand: {$this->brand} <br> Model: {$this->model} ";
 }
}